docker/credentials/store.py

```diff
@@ -75,11 +75,7 @@ def _execute(self, subcmd, data_input):
         output = None
         env = create_environment_dict(self.environment)
         try:
-            if six.PY3:
```

Comment:
@hugovk Why changing the conditional order instead changing only `six.PY3` -> `not six.PY2`?

Reply:
I think this is more readable:
```python
if thing:
    a
else:
    b
```

Than:
```python
if not thing:
    a
else:
    b
```